File: Python/231. Power of Two.py
class Solution:
    def isPowerOfTwo(self, n: int) -> bool:
        if n == 1 : return True
        if n % 2 != 0 or n <= 0: return False
        while n > 2:
            if n % 2 == 1:return False
            n /= 2
        return True

File: Java/231. Power of Two.java
class Solution {
    public boolean isPowerOfTwo(int n) {
        if(n <= 0) return false;
        while(n > 0){
            if(n == 1){return true;}
            if(n % 2 == 0){n /= 2;}
            else{return false;}
        }
        return true;
    }
}
